web/annotation_editor_layer_builder.css

```
@@ -332,3 +366,77 @@
     }
   }
 }
+
+.altText {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: auto;
+  height: 24px;
+  min-width: 88px;
+  position: absolute;
+  bottom: 8px;
+  left: 8px;
+  z-index: 1;
+  pointer-events: all;
+
+  color: var(--alt-text-fg-color);
+  font: menu;
+  font-size: 12px;
+  font-style: normal;
```

Is it actually correct to define `font` with a single argument? Looking at https://developer.mozilla.org/en-US/docs/Web/CSS/font#syntax it seems that you need *at least* two.

Could we just change this to the following?
```suggestion
  font: normal 12px menu;
```

---

For system fonts the format is `font: foobar`, here `menu` isn't a font-family.